planning/behavior_path_planner/src/utils/lane_change/utils.cpp

```diff
@@ -1065,8 +1065,7 @@ ExtendedPredictedObject transform(
   const auto & check_at_prepare_phase =
     lane_change_parameters.enable_prepare_segment_collision_check;
   const auto & prepare_duration = common_parameters.lane_change_prepare_duration;
-  const auto & velocity_threshold =
-    lane_change_parameters.prepare_segment_ignore_object_velocity_thresh;
+  const auto & velocity_threshold = lane_change_parameters.stop_velocity_threshold;
```

Comment:
Since `velocity_threshold` is also used to ignore the object consideration in the prepare segment, we should keep this variable as this is.